pages/blog/july-december-2021-at-asyncapi.md

```diff
@@ -0,0 +1,171 @@
+
+Before, you could use [AsyncAPI Playground](https://playground.asyncapi.com/) but soon we'll start redirecting traffic to new, beautiful :point_right: [AsyncAPI Studio](https://studio.asyncapi.com/) :point_left:
+
+The new Studio has a bright future, and you should expect lots of new features coming there. I definitely recommend checking out [Studio's GitHub repository](https://github.com/asyncapi/studio/).
+
+The cool stuff is that it is also integrated with [AsyncAPI CLI](https://www.asyncapi.com/tools/cli):
+```bash
+#install CLI
+npm install -g @asyncapi/cli
+#create new file using an example
+asyncapi new --file-name=asyncapi.yml --example=default-example.yaml --no-tty
+#start studio that picks up asyncapi.yml file from the context it runs in
+asyncapi start studio
+```
+
+Repository with [Playground source code](https://github.com/asyncapi/playground) will most probably be archived and moved to [organization with old archived projects](https://github.com/asyncapi-archived-repos). 
+
+## Hackathon
+
+This year, we hosted our first [AsyncAPI Hackathon](https://www.asyncapi.com/blog/hackathon-faq). Even though we did not have a dedicated marketing campaign, and even though AsyncAPI is not trivial and requires experience to build tools for it, we received 9 submissions.
+
+[Souvik De](https://www.linkedin.com/in/souvik-de-a2b941169) won first prize with [AsyncAPI Bundler](https://github.com/asyncapi/community/discussions/128). Check out Souvik talking about the AsyncAPI Bundler at AsyncAPI Conference.
+
+<YouTube id="j_KLiCRW3t0" /> 
+
+We also had 2 folks that got into the 2nd place:
+- [Ace](https://www.twitter.com/_acebuild) submitted [AsyncAPI Blocks](https://github.com/asyncapi/community/discussions/149)
+- [Greg Meldrum](https://www.linkedin.com/in/greg-meldrum-8b15a03/) submitted [Event Discovery Agent](https://github.com/asyncapi/community/discussions/147)
+
+Congrats to all of you folks!
+
+Check [this discussion to get more details on the voting process](https://github.com/asyncapi/community/discussions/165).
+
+PS. During Hackathon we also had lots of people engaging with us because of Hacktoberfest. We got [30 issues resolved and merged](https://docs.google.com/spreadsheets/d/17YYe-ZompcuUACfCGa8Pc3XunFfWRBrNllIBChJ0yWw/edit#gid=0).
+
+## Conference
+
+COVID19 is still here. Thus, we decided that we will have another AsyncAPI Conference this year only in online mode. It was a three-day event with lots of great content, and most importantly, with 1st day dedicated to contributors only.
+
+The conference was live-streamed. Full-day recordings were released immediately, and individual talks recordings will follow. All listed under [dedicated Conference playlist](https://www.youtube.com/playlist?list=PLbi1gRlP7pijq9F5eYsJomWc7Zf6EYVTZ).
+
+Till today this is how many people watched it during the conference and after as recordings:
+- Day 1:
+  - YouTube: 1119
+  - Twitch: 89 (during event only, later Twitch removes old videos)
```

Comment:
FYI, Twitch doesn't have to delete your videos, you can change that setting and **enable your account to save past broadcasts.**

![image](https://user-images.githubusercontent.com/19964402/146125711-ff5bffe8-5a17-4898-9c8c-9a0ad2febce1.png)


> "​To enable archiving, first visit your Stream Settings. You can navigate to this page from your Creator Dashboard by expanding the Settings tab on the left and then clicking Stream. Under the VOD Settings section, toggle the slider next to Store past broadcasts to enable or disable archiving."

**Source:** https://help.twitch.tv/s/article/video-on-demand

Reply:
We have these settings but you see that tooltip says it will hold them for 14days only

<img width="1150" alt="Screenshot 2021-12-15 at 09 57 29" src="https://user-images.githubusercontent.com/6995927/146155079-d8acc192-02a4-4798-b86a-3746a63558eb.png">

Reply:
@derberg Yes, but did you read my entire comment and the resource article? :)

It tells you what to do so that they are not automatically deleted after 14 days. This is a common setting that all Twitch users have to worry about. AWS had to do this stuff too.

Reply:
you mean `Highlight` option? yeah, tried, looks very manual. Also becoming partner would mean we really have to stream a lot there. Anyway, thanks for the info, will benefit in the future